app/src/main/java/com/nitramite/courier/DHLExpressStrategy.java

```diff
@@ -64,9 +63,13 @@ public ParcelObject execute(String parcelCode) {
 
                 // Parse all package related normal data found
                 if (jsonChildNode.has("delivery")) {
-                    parcelObject.setPhase(jsonChildNode.getJSONObject("delivery").getString("status").toUpperCase(Locale.getDefault())); // Phase
+                    String phase = jsonChildNode.getJSONObject("delivery").getString("status").toUpperCase(Locale.getDefault());
+                    if (phase.equals(PhaseNumber.PHASE_TRANSIT)) {
+                        phase = PhaseNumber.PHASE_IN_TRANSPORT;
+                    }
+                    parcelObject.setPhase(phase); // Phase
                 } else {
-                    parcelObject.setPhase("TRANSIT"); // Set as transit phase since it's still coming
+                    parcelObject.setPhase(PhaseNumber.PHASE_IN_TRANSPORT); // Set as in transport phase since it's still coming
```

Comment:
This comment is unintentionally comical. Better change it to "as it's still on its way" :slightly_smiling_face:.